listings/ch02-basic-concepts/random.mun

```diff
@@ -0,0 +1,5 @@
+extern fn random() -> int;
+
+pub fn random_bool() -> bool {
+    random() % 2 == 0
+}
```

Comment:
The dreaded no-end-of-line 😬 

Also, for consistency it might be good to name the files using the `listing{NUMBER}` syntax that I also used for chapters 1 & 3. I decided on this naming because someone who wants to edit a listing in the rendered markdown, doesn't per se know the connection between a file name and the particular listing unless they look at the source, whereas they can derive the name from the listing number noted underneath a Listing.

On that topic. Your listings don't add the Listing 02-0X notation. Could you add that? It also makes it a little easier to refer to listings.